data/mods/CRT_EXPANSION/deadspace/deadspaceitems.json

```
@@ -15,7 +15,15 @@
     "coverage": 0,
     "encumbrance": 0,
     "warmth": 0,
-    "artifact_data": { "charge_type": "ARTC_NULL", "effects_worn": [ "AEP_ALL_UP" ] },
+    "//": "Legacy artifact data is set for all stats +2",
+    "relic_data": {
+      "passive_effects": [
+        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "STRENGTH", "add": 2 } ] },
+        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "INTELLIGENCE", "add": 2 } ] },
+        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "PERCEPTION", "add": 2 } ] },
+        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "DEXTERITY", "add": 2 } ] }
```

```suggestion
        { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "STRENGTH", "add": 2 }, { "value": "INTELLIGENCE", "add": 2 }, { "value": "PERCEPTION", "add": 2 }, { "value": "DEXTERITY", "add": 2 } ] }
```
I just now realized that you can do this instead (linting not guaranteed) for every single one of these items.